app/src/main/java/fr/free/nrw/commons/description/DescriptionEditActivity.kt

```diff
@@ -60,17 +67,30 @@ class DescriptionEditActivity : BaseActivity(), UploadMediaDetailAdapter.EventLi
 
     private val REQUEST_CODE_FOR_VOICE_INPUT = 1213
 
+    private var descriptionAndCaptions: ArrayList<UploadMediaDetail>? = null
+
+    @Inject lateinit var descriptionEditHelper: DescriptionEditHelper
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         binding = ActivityDescriptionEditBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
         val bundle = intent.extras
-        val descriptionAndCaptions: ArrayList<UploadMediaDetail> =
-            bundle!!.getParcelableArrayList(LIST_OF_DESCRIPTION_AND_CAPTION)!!
-        wikiText = bundle.getString(WIKITEXT)
-        savedLanguageValue = bundle.getString(Prefs.DESCRIPTION_LANGUAGE)!!
+
+        if(savedInstanceState!=null){
```

Comment:
Need some time to test on a device, but I skimmed through the code. Just a few suggestions based on the [style guide](https://github.com/commons-app/commons-app-documentation/blob/master/android/Code-style.md) the app follows: 
- space after `if`
- space before and after `!=`
- space after `)`